src/iop/liquify.c

```diff
@@ -1666,14 +1666,7 @@ void init(dt_iop_module_t *module)
   // We don't use dt_iop_default_init(module); as some type int8, floatcomplex are not
   // supported by introspection.
 
-  // module is disabled by default
-  module->default_enabled = 0;
-  module->params_size = sizeof(dt_iop_liquify_params_t);
-  module->gui_data = NULL;
-
-  // all allocated to 0, which is the default
-  module->params = calloc(1, module->params_size);
-  module->default_params = calloc(1, module->params_size);
+  dt_iop_default_init(module);
```

Comment:
If you are not adding anything to `dt_iop_default_init` you can just completely delete the whole `init` and it will fall back to the default. That should work here.